### DISCUSSION TITLE
MediaPipeLiveLink Component missing

### DISCUSSION THREAD

**Discussion by SashaBoraBora** [endink/Mediapipe4u-plugin#110] [Q&A]
Hey, I am missing the MediaPipeLiveLink Component in my actor.
Also in which BP are you here? 

https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/mocap/quick_start/start_mocap.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp

**Comment by endink** [comment]
Read this:

https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/mocap/features/face_link_actor.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp

**MediapieFaceLinkActor** is an Actor, not a Component, search it in content browser and drag into your level.

**Comment by SashaBoraBora** [comment]
I am asking about this here:
https://opensource-labijie-com.translate.goog/Mediapipe4u-plugin/mocap/quick_start/prepare_components.html?_x_tr_sl=zh-CN&_x_tr_tl=en&_x_tr_hl=zh-CN&_x_tr_pto=wapp

**Reply by endink** [reply]
sorry, this component has been replaced by FaceLinkActor, the documentation here has not been updated, sorry again.
